tests/admin_inlines/tests.py

```diff
@@ -1850,10 +1852,11 @@ def test_delete_inlines(self):
             "form#profilecollection_form tr.dynamic-profile_set#profile_set-2", 1
         )
 
+    @screenshot_cases(["desktop_size", "mobile_size", "rtl", "dark"])
```

Comment:
Now that https://github.com/django/django/pull/18080 is merged, please rebase and add "high_contrast" to all added screenshot_cases ⭐

Reply:
Thank you